blocks/library/gallery/index.js

```diff
@@ -136,6 +144,12 @@ registerBlockType( 'core/gallery', {
 						<p>{ __( 'Image galleries are a great way to share groups of pictures on your site.' ) }</p>
 					</BlockDescription>
 					<h3>{ __( 'Gallery Settings' ) }</h3>
+					<SelectControl
+						label={ __( 'Link to' ) }
+						selected={ linkto }
+						onBlur={ setLinkto }
```

Comment:
Any reason to not have `onChange={ setLinkto }` here?  Currently, after changing the value of this element, you must click off the select drop down to trigger the `onBlur` before you can update the post:

![no-on-change](https://user-images.githubusercontent.com/22080/28897514-f8586d28-7795-11e7-85b0-2ddea226e9dc.gif)